src/Illuminate/Mail/Markdown.php

```diff
@@ -69,9 +69,27 @@ public function render($view, array $data = [], $inliner = null)
         $contents = $bladeCompiler->usingEchoFormat(
             'new \Illuminate\Support\EncodedHtmlString(%s)',
             function () use ($view, $data) {
-                return $this->view->replaceNamespace(
-                    'mail', $this->htmlComponentPaths()
-                )->make($view, $data)->render();
+                EncodedHtmlString::encodeUsing(function ($value) {
+                    $replacements = [
+                        '[' => '\[',
+                        '<' => '&lt;',
+                        '>' => '&gt;',
+                    ];
+
+                    $html = str_replace(array_keys($replacements), array_values($replacements), $value);
```

Comment:
It might make sense to add the JavaScript equivalent of `Object.entries()`, respectively, the inverse of `array_combine()` to the `Arr` helper, so one could do 🤔 
```suggestion
                    [$keys, $values] = Arr::entries($replacements);
                    $html = str_replace($keys, $values, $value);
```